framework/src/main/java/org/apache/drill/test/framework/CmdParam.java

```diff
@@ -55,6 +55,9 @@
     @Parameter(names = {"-rf", "--runFailed"}, description = "Run tests tagged as failures", required=false)
     public boolean runFailed  = false;
 
+    @Parameter(names = {"-sR", "--skipRandom"}, description = "Skip check for random failures", required=false)
+    public boolean skipRandom  = false;
```

Comment:
So it does not skip by default, correct? I got confused the first time i saw this. Only if the option is provided, then we skip.

Reply:
Correct.  If the option is provided, then we skip.